datadog/table_datadog_host.go

```diff
@@ -0,0 +1,101 @@
+package datadog
+
+import (
+	"context"
+	"fmt"
+
+	datadog "github.com/DataDog/datadog-api-client-go/api/v1/datadog"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableDatadogHost(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "datadog_host",
+		Description: "A host is any piece of infrastructure that runs an instance of the Datadog Agent such as a bare metal instance or a VM.",
+		// There is no Datadog hosts endpoint for getting a single host so instead we use a filter to perform an exact match
+		Get: &plugin.GetConfig{
```

Comment:
Since we already have the `name` column handled as an optional qual in the `ListConfig`, I think we can remove the `GetConfig` for now.